mediorum/ddl/ddl.go

```diff
@@ -73,71 +68,6 @@ func md5string(s string) string {
 	return hex.EncodeToString(hash[:])
 }
 
-func migrateShardBucket(db *sql.DB, bucket *blob.Bucket) {
```

Comment:
seems unrelevant to this change, but lgtm

Reply:
unrelevant is right, but straightforward cleanup never hurt (unless i just jinxed it 🤷 )